docs/src/@primer/gatsby-theme-doctocat/components/head.js

```diff
@@ -19,11 +20,13 @@ function Head(props) {
 
   let primerViewComponentsSrc
 
-  if(process.env.NODE_ENV === 'development') {
-    primerViewComponentsSrc = "http://localhost:4000/assets/primer_view_components.js"
-  } else {
-    primerViewComponentsSrc = "https://unpkg.com/@primer/view-components@latest/app/assets/javascripts/primer_view_components.js"
-  }
+  primerViewComponentsSrc = "/primer_view_components.js"
+  console.log("THIS IS THE VERCEL ENV**************", process.env.VERCEL_ENV)
```

Comment:
We prob want some condition here to use the local JS file in `process.env.NODE_ENV === 'development'` OR in a vercel preview environment (how?) OR in codespaces (how?). 

Otherwise on production run of docs site, we want to use the published package.

Reply:
I wonder if there's a need at all to point to the published package. The production doc site will be out of sync and not have the latest JS until there's a release.